SERVER/src/env_manager.py

```
@@ -0,0 +1,63 @@
+import os
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)